src/components/HorizontalTab/index.tsx

```diff
@@ -72,7 +72,7 @@ const HorizontalTab = ({selectedTab, onSelectedTabChange, tabs = []}: Horizontal
             index === selectedTab
               ? [
                   styles.activeTabItem,
-                  {borderBottomColor: index === 0 ? colors.darkBlue : colors.black}
+                  {borderBottomColor: index === 0 ? colors.darkBlue : colors.anon_primary}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change seems to be introducing a new color for the bottom border of the active tab item. However, it's not clear why the color should be different when `index === 0`. If this is intentional and there's a specific reason for it, consider adding a comment explaining why. If not, you might want to reconsider this logic.
```diff
- {borderBottomColor: index === 0 ? colors.darkBlue : colors.anon_primary}
+ {borderBottomColor: colors.anon_primary}
```
This will make the borderBottomColor consistent across all tabs.


<!-- This is an auto-generated comment by OSS CodeRabbit -->